browser-test/src/support/admin_programs.ts

```diff
@@ -1142,6 +1196,35 @@ export class AdminPrograms {
     expect(blockText).toContain(answer)
   }
 
+  async expectApplicationAnswersForBulkStatus(
+    blockName: string,
+    questionName: string,
+    answer: string,
+  ) {
+    const blockText = await this.page
+      .locator(this.selectApplicationBlock(blockName))
+      .innerText()
+
+    expect(blockText).toContain(questionName)
+    expect(blockText).toContain(answer)
+  }
+
+  async expectApplicationAnswerLinksForBulkStatus(
+    blockName: string,
+    questionName: string,
+  ) {
+    expect(
+      await this.page
```

Comment:
These should be `await expect`

Reply:
Everytime time I change the order, bin/fmt removes the await from '"await expect ..."

Reply:
ohh I see. I think this is because whatever we get back from the expect isn't a promise, at that point.

It would be slightly nicer if `selectApplicationRowForApplicant()` returned a Locator itself, instead of just a string that's later used in a locator.

As it is, I think this assert, and similar ones, need to be
`await expect(this.page.locator(selectApplicationRowForApplicant(applicant))).not.toHaveText(statusString)`

Playwright suggests using the `.toHaveText()` method on the locator, rather than the innerText method on the page:
- https://playwright.dev/docs/api/class-locator#locator-inner-text
- https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

Reply:
Hi @avaleske  - The check is to test for text.contains(xxx) the .toHaveTest() is failing saying the text doesn't match. I think toHaveText() checks for the whole text and toContain() tests for a xxxx continous charaters in a given string. 

Console output- 
 `Error: Timed out 5000ms waiting for expect(locator).toHaveText(expected)

    Locator: locator('.cf-admin-application-block-card:has-text("Screen 3")')
    Expected string: "fileupload-q"
    Received string: "Screen 3third descriptionfileupload-q-- file.txt UPLOADED (click to download) --Answered on 2024/12/06"
    Call log:`

If I change to toContain - If is failing as below- 
` Error: expect(received).toContain(expected) // indexOf

    Expected value:  "fileupload-q"
    Received object: {"_frame": {"_guid": "frame@3b741be88e6f6c86f24e257df25e8a30", "_type": "Frame"}, "_selector": ".cf-admin-application-block-card:has-text(\"Screen 3\")"}

       at support/admin_programs.ts:1214

      1212 |     await expect(
      1213 |        this.page
    > 1214 |         .locator(this.selectApplicationBlock(blockName))).toContain(questionName)
           |                                                           ^
`

Reply:
.ContainText() works!